packages/wallet/.eslintrc.js

```diff
@@ -0,0 +1,7 @@
+const path = require('path');
+module.exports = {
+  "extends": "../../.eslintrc.json",
+  "parserOptions": {
+    "project": path.resolve(__dirname, "../../tsconfig.lint.json")
+  }
+}
```

Comment:
Nit. New line at EOF. 

```suggestion
}

```